pkgs/by-name/sh/shadered/package.nix

```
@@ -38,6 +38,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./install_path_fix.patch
+    ./0001-treewide-add-missing-includes.patch
```

```suggestion
    (fetchpatch2 {
      url = "https://github.com/dfranx/SHADERed/commit/f9c44368e0fb6f5b6823e1347fcc1a1b680e00aa.patch?full_index=1";
      name = "missing-includes.patch";
      hash = "sha256-2SBU5AV3Gcm21jdHJLhgnHrLJfJ3KwF3Dsu/Zivhc3s=";
    })
```
This patch is already in PRs upstream and it's not specific to nixpkgs so maybe it's better to just pull it instead of including in tree